test/ui/conf.js

```diff
@@ -0,0 +1,14 @@
+exports.config = {
```

Comment:
It seems it's a convention to call protractor tests `e2e`, not `ui`.

Reply:
renamed